### ISSUE TITLE
AWS Account/Service Check Class

### ISSUE THREAD

**Issue by brifordwylie** [SuperCowPowers/workbench#9]
I'd like a class that tests out the current account credentials and gives some nice informative messages when it's isn't set up correctly.

We'll use a Python class obviously but it will do some superset of the following:
-  aws sts get-caller-identity
-  aws iam get-account-authorization-details
- Service Tests (note this might be just calling the meta data class)
  - S3, Data CataLog, Athena, Feature Store, Model Registry, Model Monitors

Basically I want SageWorks to ^know^ that the clients AWS account is 'good'... we can run a simple 'account check' script and we know that we have all the account + roles + policies setup. I don't want to 'find out later' that we need to add roles/policies/etc. I want to find out ^now^ :)
